### DISCUSSION TITLE
[Enhancement] [select] The select tab is displayed as a pop-up window（选项卡以弹出框的形式展示）

### DISCUSSION THREAD

**Discussion by SwankyTigerYY** [element-plus/element-plus#11162] [Feature Request]
Feature type: **`Enhancement`**

## Details

- Enhanced component name: `select`

### Features (特性）
1. Options appear as pop ups（选项以弹出框的形式出现）
2. It is better to achieve both mobile and PC（最好是移动和PC都能实现）

### Living examples （已有的例子）
quasar + q-select + showPopup()
https://v1.quasar.dev/vue-components/select

### Additional comments

quasar在移动浏览器下完美的实现了该功能，可是它在PC端并没有。我想用electron打包H5页面，但是我没有找到electron打包的应用默认为手机浏览器模式，select功能不能实现我想要的效果。最后我自己实现的该效果。我希望element能实现这个功能。

**Comment by kooriookami** [comment]
Element Plus目前还是一款桌面端UI库，不会特意去适配移动端。
